### PR TITLE
header mobile

### PR DESCRIPTION
**Pull request by zanderol**

### CODE REVIEW

File: src/partials/header.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<header class="header container">
+  <div class="header__container">
+    <a href="#" class="header__logo">
+      <img
+        src="./images/logo/Logo-l-min.png"
```
Comment (YuliiaDikun) [inline src/partials/header.html]: Please, use example.html to find out the right way in the src attribute.

File: src/partials/header.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        alt="Головний логотип сайту - морозиво"
+        class="header__logo"
+      />
+    </a>
+    <!-- <div class="header__logo">main logo</div> -->
+  </div>
+
+  <button class="menu-toggle" aria-expanded="false" aria-controls="mobile-menu">
+    <svg class="icon-burger" width="32" height="32" fill="currentColor">
+      <use href="./images/icons.svg#icon-menu__burger"></use>
```
Comment (YuliiaDikun) [inline src/partials/header.html]: Same mistake in `href` attribute.

File: src/partials/header.html (hunk 1 above)
Comment (zanderol) [reply src/partials/header.html]: got it! done!

**Review by YuliiaDikun — APPROVED** [review]

**Comment by YuliiaDikun** [comment]
Greate work!

**Pull request merged**